src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -263,6 +279,28 @@ public MeasureObjectCommand()
 
         #endregion Common parameters in both sets
 
+        /// <summary>
+        /// Set to true if Standard Deviation is to be returned
+        /// </summary>
+        /// <value></value>
```

Comment:
Please remove the line.

Reply:
Lines like 285 with empty value tags?

Reply:
Yes, empty value tags don't contain useful information.

Reply:
remove this `/// <value></value>` line as discussed.